ui/app/templates/components/wizard/tutorial-idle.hbs

```diff
@@ -4,14 +4,9 @@
 ~}}
 
 <WizardContent @headerText="Welcome to Vault" @glyph="tour" @class="collapsed" @hidePopup={{true}}>
-  <button type="button" class="button is-transparent icon dismiss-collapsed" {{action @onDismiss}}>
```

Comment:
If this is the only place we used `dismiss-collapsed` can we delete the css class too?